packages/ndla-accordion/src/AccordionSection.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { FC, Children, useState, useEffect } from 'react';
+import { AccordionBar, AccordionPanel } from './';
+
+export interface Props {
+  id: string;
+  title: string;
+  hasError?: boolean;
+  startOpen?: boolean;
+  tiny?: boolean;
+  barChildren?: React.ReactNode;
+  open?: boolean;
+  onClick?: () => void;
+}
+
+const AccordionSection: FC<Props> = props => {
+  const { id, children, startOpen, open, onClick } = props;
+  const { title, barChildren, tiny, hasError } = props; // bar props
```

Comment:
skjønte ikke helt hvorfor de er to her, hva er _bar_?

Reply:
AccordionBar